11ty/definitions/underrepresented-minority.md

```diff
@@ -0,0 +1,47 @@
+---
+title: underrepresented minority (URM)
+slug: underrepresented-majority
+defined: true
+speech: noun
+flag:
+  level: avoid
+reading:
+  - text: Free Dictionary definition
+    href: https://medical-dictionary.thefreedictionary.com/underrepresented+minority
+---
+
+group whose representation is disproportionately less than their proportion in the general population; often racial or ethnic minority is implied, but can be used to represent any identifier, such as LGBTQIA+, disabled, gender, etc.
+
+abbreviated often to URM
+
+## Issues
+
+Often people use URM to mean "minoritised racial group", when it doesn't explicitly mean racial group in all contexts (even if that's the most commonly used context). It can cause confusion when which minoritised group is being spoken of, and can also lessen/erase additional intersections of identity. For example, if someone is disabled and of a minoritised racial group, using URM to imply racial group can erase their disability.
+
+Using a term like URM can also erase meaningful specificity. For example, an issue might specifically be relevant to Black folks, but by referring to the group as URM, it erases their Blackness, conflates/waters down issues, and reinforces the power dynamic of white and everyone else.
+
+URM can also imply that the group in question isn't "showing up" to be represented; that the burden is on the URM group to show up or to do more to improve their numbers and representation. Meanwhile, the complement, the overrepresented majority, tends to be unnamed.
+
+## Impact
+
+By naming the underrepresented minority often without the overrepresented majority, we are often reinforcing the idea that people harmed by oppressive systems are on the hook to fix them.
+
+By repeatedly referring to groups as "[minorities](minorities)," they are being infantilised to their numerical representation and made to feel minor. The vagueness also abstracts the identify of the marginalised group, making it difficult to address why they are minoritised to begin with and often erasing the complexity of those intersections.
+
+## Usage
+
+When using any term for a group, it's important to contextualise it into people. So, for example, "people from the underrepresented minorities are less likely to be affected by this change" over "the overrepresented majority are less likely to be affected by this change."
```

Comment:
Given this paragraph from the `overrepresented majority` page (the other file): 

>When using any term for a group, it's important to contextualise it into people. So, for example, "people from the overrepresented majority are less likely to be affected by this change" over "the overrepresented majority are less likely to be affected by this change."

Should this be updated to `people from the overrepresented majority are less likely [...]` rather than `the overrepresented majority are less likely [...]`?